metrics/examples/grafana/grafana.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: grafana
-        image: strimzilab/grafana-openshift:latest
+        image: grafana/grafana:latest
         imagePullPolicy: IfNotPresent
```

Comment:
Sorry, I know this is not changed in this PR. But do we really want to have this here? I would prefer to use the default (i.e. leave this line out).